coveralls.json

```diff
@@ -3,7 +3,8 @@
     "minimum_coverage": 30
   },
   "skip_files": [
-    "elixir_boilerplate/release/migration.ex",
+    "elixir-boilerplate/release_tasks.ex",
+    "mix/tasks/erlang.check_version.ex",
     "test/support"
```

Comment:
If we have `test/support` here, shouldn’t the two other paths begin with `lib`? 🤔

Reply:
I had the same thought and it seems like `lib/` is optional. I’ll add it anyway to remove some magic.